docs/adrs/024-remote-module-invocation.md

```diff
@@ -45,14 +45,17 @@ model_management:
             config:
                 connection:
                     hostname: str <remote host>
-                    port: int <remote port>
-                    protocol: Optional[str]="grpc" <protocol the remote server is using (grpc or http)>
+                    port: Optional[int]=80/443 <remote port>
                     tls:
                         enabled: Optional[bool]=False <if ssl is enabled on the remote server>
                         ca_file: Optional[str]=None <path to remote ca file>
                         cert_file: Optional[str]=None <path to MTLS cert>
                         key_file: Optional[str]=None <path to MTLS key>
+                        insecure_verify: Optional[bool] = False <if server's cert should be verified>
                     options:  Optional[Dict[str,str]]={} <optional dict of grpc or http configuration options>
+                    timeout: Optional[int]=60 <optional client timeout setting>
+                protocol: Optional[str]="grpc" <protocol the remote server is using (grpc or http)>
```

Comment:
Interesting! I didn't realize this used to be under `connection`. Why did you move it out?

Reply:
This change revolves around my idea to try to make `ConnectionInfo` a generic dataclass that wasn't tied to this implementation.